lib/active_merchant/billing/gateways/pac_net_raven.rb

```diff
@@ -148,9 +148,9 @@ def fraud_review?(response)
 
       def success?(response)
         if %w(cc_settle cc_debit cc_preauth cc_refund).include?(response[:action])
-          !response['ApprovalCode'].nil? and response['ErrorCode'].nil? and response['Status'] == 'Approved'
+          !response['ApprovalCode'].nil? && response['ErrorCode'].nil? && (response['Status'] == 'Approved')
         elsif response[:action] = 'void'
-          !response['ApprovalCode'].nil? and response['ErrorCode'].nil? and response['Status'] == 'Voided'
+          !response['ApprovalCode'].nil? && response['ErrorCode'].nil? && (response['Status'] == 'Voided')
```

Comment:
I like the addition of the parentheses to more clearly show these are three conditional statements.